caddy/php-server.go

```diff
@@ -47,11 +47,14 @@ For more advanced use cases, see https://github.com/dunglas/frankenphp/blob/main
 			cmd.Flags().StringP("root", "r", "", "The path to the root of the site")
 			cmd.Flags().StringP("listen", "l", "", "The address to which to bind the listener")
 			cmd.Flags().StringArrayP("worker", "w", []string{}, "Worker script")
-			cmd.Flags().StringArrayP("watch", "", []string{}, "Directory to watch for file changes")
+			cmd.Flags().StringArray("watch", []string{}, "Glob pattern of directories and files to watch for changes")
 			cmd.Flags().BoolP("access-log", "a", false, "Enable the access log")
 			cmd.Flags().BoolP("debug", "v", false, "Enable verbose debug logs")
 			cmd.Flags().BoolP("mercure", "m", false, "Enable the built-in Mercure.rocks hub")
-			cmd.Flags().BoolP("no-compress", "", false, "Disable Zstandard, Brotli and Gzip compression")
+			cmd.Flags().Bool("no-compress", false, "Disable Zstandard, Brotli and Gzip compression")
+
+			cmd.Flags().Lookup("watch").NoOptDefVal = "\"" + defaultWatchPattern + "\""
```

Comment:
```suggestion
			cmd.Flags().Lookup("watch").NoOptDefVal = `"` + defaultWatchPattern + `"`
```

Why are the quotes needed?

Reply:
This is an artifact from the StringSlice try, which uses comma as separator and the default pattern actually contains comma. It's now removed :+1: